src/realmGeolBasin.ttl

```diff
@@ -96,25 +101,49 @@ soreagb:PeripheralForelandBasin rdf:type owl:Class ;
 
 ###  http://sweetontology.net/realmGeolBasin/RetroarcForelandBasin
 soreagb:RetroarcForelandBasin rdf:type owl:Class ;
-                            rdfs:subClassOf soreagb:Basin ;
+                            rdfs:subClassOf soreagb:SedimentaryBasin ;
                             rdfs:label "retroarc foreland basin"@en .
 
 
+###  http://sweetontology.net/realmGeolBasin/Subbasin
+soreagb:Subbasin rdf:type owl:Class ;
+                     dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                     rdfs:subClassOf soreagb:SedimentaryBasin ;
+                     skos:altLabel "sub-basin"@en ;
+                     skos:definition "A sedimentary basin may be subdivided into two or more subbasins that have separate depocentres and are separated from one another by some tectonic/structural element, such as a fault or basement high. (Definition created here: Paul Blake, John McKellar)"@en ;
+                     skos:prefLabel "subbasin"@en .
+
+
 ###  http://sweetontology.net/realmGeolBasin/SuccessorBasin
 soreagb:SuccessorBasin rdf:type owl:Class ;
-                     rdfs:subClassOf soreagb:Basin ;
+                     rdfs:subClassOf soreagb:SedimentaryBasin ;
                      rdfs:label "successor basin"@en .
 
 
+###  http://sweetontology.net/realmGeolBasin/Superbasin
+soreagb:Superbasin rdf:type owl:Class ;
+                     dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                     rdfs:subClassOf soreagb:SedimentaryBasin ;
+                     skos:definition "A superbasin is a hierarchial term applied to  a group of largely interconnected or associated sedimentary basins of the same or similar age. Its component basins may be separated or partially separated by tectonic elements that define 'boundaries'  between them, or they may have been subjected to somewhat different geological influences. In the Australian Mesozoic, the Great Australian Superbasin, composed of the Nambour, Clarence-Moreton, Surat, Eromanga, Carpentaria and other basins, is a prime example, variously covering a geographically-extensive area in parts of Queensland, the Northern Territory, New South Wales and South Australia. (Definition created here)"@en ;
+                     skos:prefLabel "superbasin"@en .
+
 ###  http://sweetontology.net/realmGeolBasin/TranspressionalBasin
 soreagb:TranspressionalBasin rdf:type owl:Class ;
-                           rdfs:subClassOf soreagb:Basin ;
+                           rdfs:subClassOf soreagb:SedimentaryBasin ;
                            rdfs:label "transpressional basin"@en .
 
 
+###  http://sweetontology.net/realmGeolBasin/Trough
+soreagb:Trough rdf:type owl:Class ;
+                     dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                     rdfs:subClassOf soreagb:SedimentaryBasin ;
```

Comment:
Trough sub-class-of SedimentaryBasin is not convincing to me. That implies that every individual Trough is also a SedimentaryBasin . Is this true @smrgeoinfo ?

Reply:
Considering basin type or status, we may have the hierarchy of superbasin, basin, sub-basin or trough. However, not every basin has a higher-order superbasin—in fact, most don’t. The preeminent superbasin in Australia is the Great Australian Superbasin, which, from a hydrogeological point of view, is intimately associated with the Great Artesian Basin (but with the latter also encompassing older aquifers of underlying basinal systems). 
And, not every basin has to have a sub-basin or a trough. In the context here, a trough is essentially a sub-basin. But, for example, in the continental realm, and for the principal Phanerozoic troughs of Australia, most, if not all, form parts of larger sedimentary basins. And, indeed, for the extensive, Sydney-Gunnedah-Bowen basin system of eastern Australia (and also considering the Cooper Basin and, to some extent, the Galilee Basin of central Australia), the formation of troughs at ≈300 Ma represents the initiation of basin formation and subsequent growth. These troughs represent the initial structural formation of a series of half-grabens through dextral (right-lateral) transtension as a thermally-elevated Pangaea switched from a clockwise to an anticlockwise rotation. This was thus a global event, and the formation of such troughs (largely magmatic grabens and half-grabens) not only occurred in the Southern Hemisphere (in Gondwana), but also in the Northern Hemisphere (in Laurussia/Laurasia).

The main troughs in the Bowen Basin fall in the basin, but there are a few that sit on its western margin (viz., the Katanga and Arbroath troughs), but they are regarded as part as part of the basin system, and also to form part of the ‘(latest Carboniferous –) Early Permian East Australian Rift System’. They could be considered to be small basins in their own right, but considering their overall geological and tectonic context, this would be stretching things a little too far. 

In the oceanic realm, a trough is regarded as an elongate depression that is differentiated from an oceanic trench by being shallower, narrower and shorter. Unlike (subduction-zone) trenches, they form through a number of geological mechanisms, including rifting. In this realm, they form part of oceanic basins, the floor of which receives sediment of one kind or another.

So, to answer the question, troughs form parts of sedimentary basins, whether continental or oceanic, and receive sediment.

Reply:
Partonymy (mereonomy) is not the same as subsumption. 
I think there is a flaw here.

Reply:
Hi Simon, your point is a little ambiguous!! If you have a problem with a trough being part of a sedimentary basin (and, as such, being a sedimentary basin in its own right), then you must also have a problem with a sub-basin (also being a sedimentary basin in its own right) being part of a sedimentary basin. Issue is raised with the former, but perplexingly not with the latter. As an example, John & Fielding (1993; https://www.publish.csiro.au/AJ/AJ92014) refer to the Denison Trough a being a deep elongate sub-basin located along the western margin of the Bowen Basin.

Reply:
Nick has advised me to provide further advice: Much of the proposed structure of the Features ontology is based on the Guide for Geological Nomenclature in Sweden (Kumpulainen, R.A.: https://doi.org/10.1080/11035897.2016.11786660. I quite like the way things are physically set out here. Nonetheless, the primary sources for this compilation are cited therein and embrace the ICS/IUGS, NACS, Norwegian and Finnish nomenclatural guides.
You will note that Kumpulainen regards a ‘trough’ as a structural entity, although the term is not listed in his Figure 1 (the template for the ontology). For comparison, Neuendorf et al. (2011), in the Glossary of Geology, define the term ‘trough’ as follows (here excluding non-applicable usage):
trough [beach] A small linear depression formed just offshore on the bottom of a sea or lake and on the landward side of a longshore bar. It is generally parallel to the shoreline, and is always under water. A trough may be excavated by the extreme turbulence of wave and current action in the zone where breakers collapse. Inappropriate syn: runnel [beach]; low [beach].

trough [geomorph] (a) Any long, narrow depression in the Earth's surface, such as one between hills; esp. a broad, elongate U-shaped valley, such as a glacial trough or a trench. (b) The channel in which a stream flows.

trough [marine geol] An elongate depression of the sea floor that is shallower than a trench, and with less steeply sloping sides.

trough [struc geol] (a) An obsolete term for a graben. (b) The lowest point of a given stratum in any profile through a fold. Cf: trough line.  <<<< forget the latter here

trough [tect] A linear depression or basin that subsides as it receives clastic material, located not far from the source supplying the sediment. Obsolete syn: geosynclinal trough.

Referring only to the marine and structural/ tectonic applications, one can argue about the definitions provided and potential overlap of the terms, but, in essence, they all embrace sedimentary basins at varying scales. In Queensland, basin terminology has encompassed the use of ‘trough’ in the sense of a ‘sub-basin’ [e.g. the Denison Trough in the Bowen Basin (as I have advised previously, above), the Koburra Trough in the Galilee Basin, Westgate Trough in the Adavale Basin, etc.] and this long-established nomenclatural usage has been widely accepted (in particular, by Geoscience Australia). These troughs are largely, if not all, half-grabens. This runs in the face of what Neuendorf et al. have indicated (see above)—that a trough is an obsolete structural term for a graben. In the cited basins, the half-graben structures were formed through tectonism, blurring the distinction between structural geology and tectonics. To some extent, the distinction between structures and tectonic units is also be blurred, and, perhaps, we need to accept this as fact, as argument will be enduring. But, please provide alternative views. Cheers ☺☺☺

Reply:
To start, note that there are three Troughs in SWEET currently: sorepsg:Trough (geometric object),  sophfd:Trough (sedimentary basin), sophfd:Trough (geometric  object with low pressure).

Reply:
Basins have parts. they might be called 'sub-basins' or 'troughs'. Some basins are called troughs.    If someone can propose an unambiguous, logically coherent definition to distinguish a sub-basin from trough, or a trough from a basin, Then we have some distinct concepts for the ontology. If not, perhaps its best to stick with 'Sedimentary basin' and 'sub-basin' with various skos:altLabels. ?

Reply:
This is a difficult one!! Most, if not all of the broadly continental troughs that I know of, are parts of basins, and are essentially subbasins. They are generally interconnected structural entities, grabens and half-grabens, that formed through extension/transtension. 

But there is one that I know of where this is not the case, and it has been renamed as a syncline—but the old name still sticks. Where they have formed as faulted structures, they can be defined by their steeply dipping sides. However, I cannot be absolutely sure that all continental troughs are subbasins and not basins in their own right.
In the oceanic realm, as previously indicated, troughs form through a variety of mechanisms and can be narrow basin or geological rifts. As such, they receive sediment and form subbasins within their larger oceanic basins, also sedimentary basins.

So, considering the above, I think that the suggestion by Stephen that troughs may be either subbasins (parts of basins) or basins in their own right should prevail. Note that, apart from low-pressure, meterologic troughs, there are also glacial troughs, another name for U-shaped glacial valleys, which, of course, are also sedimentary basins (and subbasins).

Precis: Let’s use basins and subbasins, with troughs as an altLabel for both subbasins and basins, but I think glacial trough can stand alone as an altLabel for a glacial valley, under a geomorphological ontology that we need not concern ourselves with(?).

Reply:
sounds like can be resolved with basin and subBasin

Reply:
Yes